editor/scene_tree_dock.cpp

```diff
@@ -1061,6 +1061,28 @@ void SceneTreeDock::_tool_selected(int p_tool, bool p_confirm_override) {
 				}
 			}
 		} break;
+		case TOOL_TOGGLE_SCENE_UNIQUE: {
+			List<Node *> selection = editor_selection->get_selected_node_list();
+			List<Node *>::Element *e = selection.front();
+			if (e) {
+				UndoRedo *undo_redo = &editor_data->get_undo_redo();
+				Node *node = e->get();
+				bool enabled = node->is_unique_name_in_owner();
+				if (!enabled && get_tree()->get_edited_scene_root()->get_node_or_null("%" + String(node->get_name())) != nullptr) {
+					accept->set_text(TTR("Another node already exists using this unique name."));
```

Comment:
The same condition is checked when toggling unique name and when renaming node, but the error messages are slightly different. I think the difference is unnecessary, and adds work to i10n.

| | Message |
| --- | --- |
| Toggle | Another node already exists using this unique name. |
| Rename | Another node already uses this unique name in the scene. |

Reply:
Alright, will be more careful with this next time. Thanks!